tests/scripts/minikube.sh

```diff
@@ -74,7 +74,7 @@ copy_image_to_cluster() {
 }
 
 # configure dind-cluster
-KUBE_VERSION=${KUBE_VERSION:-"v1.7.5"}
```

Comment:
is this released already? I checked and it wasn't released. 
You can do a `minikube get-k8s-versions` and see if v1.8.0 appears